todo_app/data/session_items.py

```diff
@@ -1,22 +1,41 @@
-from flask import session
-
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
+from todo_app.data.item import Item
+import requests
+import os
 
 def custom_sort(t):
     return t['status']
 
+def get_lists():
+    """
+    Gets all lists on a board 
+    """
+    params = {
+        'key': os.environ['ACCOUNT_KEY'],
+        'token': os.environ['SECRET_KEY']
+    }
+    response = requests.request('GET', 'https://api.trello.com/1/boards/607f1ee7c99b117ea947851a/lists', params=params)
+    listMap = {}
+    for r in response.json():
+        listMap[r['id']] = r
+    return listMap
+
 def get_items():
     """
     Fetches all saved items from the session.
 
     Returns:
         list: The list of saved items.
     """
-    return sorted(session.get('items', _DEFAULT_ITEMS), key = custom_sort, reverse=True)
-
+    #return sorted(session.get('items', _DEFAULT_ITEMS), key = custom_sort, reverse=True)
```

Comment:
If we're not using this any more then let's delete it rather than just commenting it out, we can always recover it with git (and by extension, the same for the custom_sort function)

Reply:
done